.cirun.yml

```diff
@@ -2,7 +2,7 @@ runners:
   - name: aws-gpu-runner
     cloud: aws
     instance_type: g4dn.xlarge
-    machine_image: ami-0678adbdcb4c3a662
+    machine_image: ami-0139a05e194cbd9a0
```

Comment:
Note: This failed here: https://github.com/scverse/anndata/pull/1084/checks?check_run_id=15487580477
Is this AMI in `eu-west-1` (that's default region)? If not, then you need to add `region: <region-name>` in here.

Reply:
I believe this didn't work because I didn't wait for the AMI to build to stop the instance, so the AMI failed to build.

Reply:
How would I tell what region it's associated with? I added eu-north-1 for the new one, but it's not super clear from the UI

Reply:
It's visible on the top right in the aws console, you already got it anyway. :)